src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -158,7 +158,7 @@ public static class EditPersonDescriptor {
         private Address address;
         private WorkExp workExp;
         private Set<Tag> tags;
-        private Interest interest;
+        private Set<Interest> interests;
```

Comment:
Good job in making the Interest field of a person as a Set! More applicable to a real world context as a person can have multiple interests!